_episodes/01-design.md

```diff
@@ -17,41 +17,49 @@ see [the instructor training course][training].
 
 ## Reverse Instructional Design
 
+### Idealized
+
 In principle,
 we design lessons in four stages:
 
-1.  **Conceptual**:
-    describe who the lesson is for,
-    what its overall goals are,
+1.  **Conteptual:** describe target audience,
+    overall lesson's goals,
     and how long it is going to be.
-    For example,
-    the lesson might be for people who have taught themselves
+
+    **Example**:
```

Comment:
`:` in boldface. See https://writing.stackexchange.com/questions/14157/do-you-bold-punctuation-directly-after-bold-text.

Reply:
Isn't this in accordance with the latest "Chicago Manual of Style".

Reply:
> A punctuation mark following emphasized text (in italics, boldface, small caps, caps, letter spacing, etc.) is also emphasized.

You used `**Conteptual:**` so I prefer to be consistent and have `**Example:**`.

Reply:
`Conceptual:` and `Example` are different --  the former is a part of a sentence, the latter -- a smallish title (without the pound sings). I didn't want to use a subtitle for the word `Example`. Shall we use a callout box for the example?

Reply:
Or remove the `:` after `Example`?